executor/joiner.go

```diff
@@ -182,6 +200,34 @@ func (j *baseJoiner) filter(input, output *chunk.Chunk, outerColsLen int) (bool,
 	return chunk.CopySelectedJoinRows(input, innerColOffset, outerColOffset, j.selected, output)
 }
 
+// filterAndCheckOuterRowStatus is used to filter the result constructed by
+// tryToMatchOuters, the result is built by multiple outer rows and one inner
+// row. The returned outerRowStatusFlag slice value indicates the status of
+// each outer row( matched/unmatched/hasNull).
```

Comment:
Please set the blackspace correctly.
`row( matched/unmatched/hasNull)` -> `row (matched/unmatched/hasNull)`